content/docs/csidriver/installation/helm/unity.md

```diff
@@ -295,8 +295,14 @@ Procedure
     At the end of the script unity-controller Deployment and DaemonSet unity-node will be ready, execute command `kubectl get pods -n unity` to get the status of the pods and you will see the following:
     
     * One or more Unity Controller (based on controllerCount) with 5/5 containers ready, and status displayed as Running.
-    * Agent pods with 2/2 containers and the status displayed as Running. 
-    
+    * Agent pods with 2/2 containers and the status displayed as Running.
+
+**Note**:
+   To install nightly or latest csi driver build using bash script use the below command:
+      `/csi-install.sh --namespace unity --values ./myvalues.yaml --version nightly/latest`
+
+8. You can also install the driver using standalone helm chart using the command `helm install --values  myvalues.yaml --namespace unity unity ./csi-unity`
+
```

Comment:
@karthikk92 , I feel Point 8 can be more descriptive and can be same as Point No 4  in PR https://github.com/dell/csi-unity/pull/82/files